### PR TITLE
Add error message to missing block

### PR DESCRIPTION
**Pull request by gterzian**
To make debugging of issues like https://github.com/massalabs/massa/issues/2535 easier.

### CODE REVIEW

File: massa-storage/Cargo.toml [hunk 1]
```diff
@@ -6,7 +6,10 @@ edition = "2021"
 
 [dependencies]
 parking_lot = "0.12"
+massa_logging = { path = "../massa-logging" }
 massa_models = { path = "../massa-models" }
+serde_json = "1.0"
+tracing = "0.1"
```
Comment (AurelienFT) [inline massa-storage/Cargo.toml:12]: Do we need tracing and json if we use massa-logging ?

Comment (gterzian) [reply massa-storage/Cargo.toml:12]: Yes, see https://github.com/massalabs/massa/issues/2543

Comment (AurelienFT) [reply massa-storage/Cargo.toml:12]: mhh ok.

**Review by AurelienFT — APPROVED** [review]

**Comment by AurelienFT** [comment]
bors merge

**Pull request merged**